### DISCUSSION TITLE
Pydantic Documentation Style – UK English vs. US English Spelling

### DISCUSSION THREAD

**Discussion by ybressler** [pydantic/pydantic#6482] [General]
_This one is simple._ Which style of English do we go with? [**_UK English_** or. **_US English Spelling_**](https://www.oxfordinternationalenglish.com/differences-in-british-and-american-spelling/)? Do we care? Or should we have a single choice we go with (and aspire to be consistent).

**Example:**
Some parts of our documentation have spelling such as `behaviour `. Others have the US version `behavior`.

**Comment by mzamini92** [comment]
Here are some of the arguments for using UK English spelling:

It is the most common style of English in the world.
It is the style of English used by the Python documentation.
It is the style of English used by the Pydantic team.
Here are some of the arguments for using US English spelling:

It is the style of English used by most software developers in the United States.
It is the style of English used by most popular libraries and frameworks in the United States.

**Reply by samuelcolvin** [reply]
I think the python documentation uses US English.

**Comment by tpdorsey** [comment]
Python documentation contains both usage, for example: behavior and behaviour, color and colour.

I don't have a strong opinion on this other than being a native US English writer. I have been standardizing on (mostly) US English usage and spelling as I work through documentation updates, but it's a work in progress.

**Comment by samuelcolvin** [comment]
This problem is entirely my fault, I live in the UK so naturally have my spell checker and brain configured for UK English.

We should generally use US English, but not the end of the day if we have a few mistakes in this regard.

Even Python's documentation has both spellings of behavior.
